packages/sanity/src/core/studio/components/navbar/resources/ResourcesButton.tsx

```diff
@@ -0,0 +1,46 @@
+import {HelpCircleIcon} from '@sanity/icons'
+import {Box, Button, Flex, Menu, MenuButton, Text, Tooltip} from '@sanity/ui'
+import React from 'react'
+import styled from 'styled-components'
+import {useColorScheme} from '../../../colorScheme'
+import {useGetHelpResources} from './helper-functions/hooks'
+import {ResourcesMenuItems} from './ResourcesMenuItems'
+
+const StyledMenu = styled(Menu)`
+  max-width: 300px;
+  min-width: 200px;
+`
+
+export function ResourcesButton() {
+  const {scheme} = useColorScheme()
+
+  const {value, error, isLoading} = useGetHelpResources()
+
+  return (
+    <Flex>
+      <Tooltip
```

Comment:
Tiny annoyance - this may not be easy to fix, so could potentially be skipped if so:
<img width="300" alt="image" src="https://github.com/sanity-io/sanity/assets/48200/a5c63161-49d8-41ad-80a8-e72c762019e7">

A pixel or two of this tooltip ("Help and Resources" title) shows below the menu when open. _Ideally_ the tooltip would be hidden once the menu is open.

Reply:
Great pickup – this UX bug is also present on the existing workspaces button.

Let's potentially address this as a fast follow in Sanity UI? I don't believe tooltips should persist on click